app/lib/cost_calculator.rb

```diff
@@ -20,13 +22,33 @@ def work_items_total(items, pricing, vat)
 
     def work_item_total(item, pricing, vat)
       rounded_cost = (item.time_spent.to_f / 60) * pricing[item.work_type] * (1 + (item[:uplift].to_f / 100)).round(2)
-      return (rounded_cost * (1 + pricing.vat)).floor(2) if vat == true
+      return (rounded_cost * (1 + pricing.vat)).floor(2) if vat
 
       rounded_cost
     end
 
+    def disbursements_total(vat, object)
+      pricing = Pricing.for(object)
+      if vat
+        disbursements_total_cost_with_vat(object, pricing).round(2)
+      else
+        disbursements(object).sum(&:total_cost_without_vat).round(2)
+      end
+    end
+
+    def disbursements_total_cost_with_vat(object, pricing)
+      disbursements(object).sum do |i|
+        vat_multiplier = i.apply_vat == 'true' ? pricing.vat : 0
+        (i.total_cost_without_vat * (1 + vat_multiplier)).floor(2)
```

Comment:
so we actually already store a VAT amount for disbursements :(

we should work out if we actually want that and use it here or remove it as not required so that we have a single source of truth

Reply:
yeah the reason I recalculated it here was because I was unsure of whether the VAT amounts stored in the table would be in line with the new calculations for summing VAT - I agree that we should have a single source of truth, thought I may leave this as it is for now until the stuff surrounding VAT is more clear